community/blog/2021-12 2021, the year that Arcus matured/README.md

```diff
@@ -0,0 +1,173 @@
+* Capability to use telemetry type filter outside Arcus-related Serilog sink
+* Capability to add public exception properties to exception telemetry
+
+See our [features docs](https://observability.arcus-azure.net/) for more information on our available functionality.
+
+## Templates
+The [templates repository](https://github.com/arcus-azure/arcus.templates) is probably the easiest place to get started with Arcus. It provides development-ready project templates for you to use at the start of your project. It already contains all the necessary best-practices and correctly use of Arcus libraries so you don't have to worry about infrastructure code but can focus on the business end.
+
+2021 was the year that we took this to another level and created project templates for Azure Functions.
+* Capability to create Azure Functions project with a HTTP endpoint trigger 
+* Capability to create Azure Functions project running a timer to log finished Databricks log runs
+
+There are already several project options and additional built-in functionality available for these new and existing project templates, so to make them more development-ready:
+* Capability to opt-in for health checks
+* Capability to track only 5xx requests in health endpoint
+* Capability to configure minimum Serilog logging level from application settings
+* Capability to use .NET project templates directly from Visual Studio 
+
+When the [messaging library](https://github.com/arcus-azure/arcus.messaging) is ready to provide the message routing outside message pumps, we create additional Azure Functions project templates with Azure Service Bus triggers. 
+See our [feature docs](https://templates.arcus-azure.net/) for more information on our available functionality.
+
+## Background Jobs
+Last but not least, we also have a new scheduled job in the [background jobs repository](https://github.com/arcus-azure/arcus.backgroundjobs). We already had a job that would notify us when an Azure Key Vault secret was expired, but now we also are able to notify consumers when a Service Principal client secrets is about to expire or is expired.
+
+This is a great way to keep track of your available client secrets. An Azure Event Grid event will notify you with all the details (secret name, expiration date). This library has already shown it's added-value by providing low-level infrastructure code that takes care of missing links between services.
+
+See our [feature docs](https://background-jobs.arcus-azure.net/) for more information on our available functionality.
+
+## Scripting
+The [scripting repository](https://github.com/arcus-azure/arcus.scripting) has received various updates. From Azure API Management to Azure SQL databases to Azure DevOps integrations. This repository has always be a place where many things come together. Repeated tasks in CI build pipelines are local development. The library was more used this year, which lead to great new ideas and functionality:
```

Comment:
```suggestion
The [scripting repository](https://github.com/arcus-azure/arcus.scripting) has received various updates. From Azure API Management to Azure SQL databases to Azure DevOps integrations. This repository has always been a place where many things come together. Repeated tasks in CI build pipelines are local development. The library was more used this year, which lead to great new ideas and functionality:
```

Reply:
I don't see what you mean with the sentence
> Repeated tasks in CI build pipelines are local development.
Maybe you mean
> Repeated tasks in CI build pipelines or local development.

But then again, it feels like something is missing ?

Reply:
Changed it like `... or for local development`. thx.